api/app/wildfire_one/wfwx_api.py

```diff
@@ -291,6 +291,15 @@ async def get_wfwx_stations_from_station_codes(
     return requested_stations
 
 
+async def get_wfwx_all_active_stations(
+        session: ClientSession,
+        header) -> List[WFWXWeatherStation]:
+    """ Return the WFWX station ids and codes that are active from WFWX API. """
```

Comment:
Do we have to filter on wildfire stations only? I'm guessing we can rely on the person adding the stations to know what the correct station is. But might be worth checking with @tessmgood ?

Reply:
What other stations are there besides wildfire stations?

Reply:
wf1 api propogates weather info for more than just stations managed by bcws. e.g. crd weather stations. Again, I don't know that it matters, since presumable the user knows what they are doing.